src/actors/redis-scanner/redis-scanner.py

```diff
@@ -0,0 +1,45 @@
+import os
+import re
+import json
+import shlex
+from subprocess import Popen, PIPE
+
+
+def _execute(cmd):
```

Comment:
same here check_output plus https://docs.python.org/2/library/subprocess.html#subprocess.CalledProcessError can be a more sexy solution here.